core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -457,4 +643,26 @@ private void workerLostHandler(long id) {
   private void registerNewWorkerConfHandler(long id, List<ConfigProperty> configList) {
     mWorkerConfigChecker.registerNewConf(id, configList);
   }
+
+  /**
+   * Sets the master id and hostname.
+   */
+  private void setMasterIdAndHostname() {
+    mMasterHostname = Configuration.get(PropertyKey.MASTER_HOSTNAME);
+    try {
+      RetryUtils.retry("get master id",
```

Comment:
Is there somewhere where this master logs its own master id?

Reply:
I am not quite sure why do the master need to log its id?

The id is just a representative of a master and it is meaningless to clients or programmers. Each time we will try to get an id for a standby master and if fails, we will assign a new id and ask the master to register with configuration. 

In addition, I do not find that we log the worker id.

Is there a need for logging the master id? Thanks!

Reply:
It would be helpful for debugging anything.

I think we log the worker id on the worker: https://github.com/Alluxio/alluxio/blob/v1.7.1/core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerProcess.java#L232

Reply:
In AlluxioMasterProcess.startMasters, when this master is a standby master, it will log:
```
LOG.info("Standby master with id {} starts sending heartbeat to leader master.", mMasterId);
```
Thanks!